src/popup/components/Payload.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    v-if="payload && payload.length"
+    class="payload"
+  >
+    <div class="payload-header">
+      <span class="tittle">Payload</span>
+      <slot />
+    </div>
+
+    <div class="payload-text">
+      {{ payload }}
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Payload',
+  props: {
+    payload: { type: String, default: '' },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/variables';
+@use '../../styles/typography';
+
+.payload {
+  @extend %face-sans-14-regular;
+
+  color: variables.$color-white;
+  width: 100%;
+  text-align: left;
+  margin-top: 16px;
```

Comment:
I suggest to set the top margin in the places where the component is used instead of making it global. Especially that this margin is overridden to 0 in some places.